### DISCUSSION TITLE
Create ScaledObject . but trigger is not happening on azure service bus count.

### DISCUSSION THREAD

**Discussion by sasingh04-fadv** [kedacore/keda#2879] [Q&A / Need Help]
Here is the output of scaledObject. Is anything wrong in below output. Because scaling is not happening ?
sanjeev@Azure:~$ kubectl get scaledobject -n prism-akscontainer
NAME                     SCALETARGETKIND      SCALETARGETNAME   MIN   MAX   TRIGGERS           AUTHENTICATION          READY   ACTIVE    FALLBACK   AGE
prism-processor-scaler   apps/v1.Deployment   qc-ga-dade        1     5     azure-servicebus   azure-servicebus-auth   False   Unknown   Unknown    4h53m

**Comment by tomkerkhove** [comment]
Would you mind sharing your configuration and KEDA logs please?
